tests/ui/simd/intrinsic/float-math-pass.rs

```diff
@@ -20,7 +21,7 @@ use std::intrinsics::simd::*;
 macro_rules! assert_approx_eq_f32 {
     ($a:expr, $b:expr) => {{
         let (a, b) = (&$a, &$b);
-        assert!((*a - *b).abs() < 1.0e-6, "{} is not approximately equal to {}", *a, *b);
+        assert!((*a - *b).abs() < 1.0e-6);
```

Comment:
Nit: Can we retain the message?

```suggestion
        assert!((*a - *b).abs() < 1.0e-6, concat!(stringify($a), " is not approximately equal to ", stringify!($b)));
```